WcaOnRails/spec/i18n_locale_files_spec.rb

```diff
@@ -0,0 +1,10 @@
+# frozen_string_literal: true
+require 'rails_helper'
+require 'i18n-spec'
+
+Dir.glob(Rails.root.join('config', 'locales', '*.yml')).each do |locale_file|
+  RSpec.describe locale_file.to_s do
+    it { is_expected.to have_valid_pluralization_keys }
```

Comment:
I'm having trouble understanding this spec. Is there anything specific to the current `locale_file` here? Is the fact that this works some magic side effect of `RSpec.describe`?